docs/shared/mental-model/large-tasks.json

```diff
@@ -1968,86 +1968,6 @@
         }
       }
     },
-    {
-      "name": "e2e-storybook-angular",
```

Comment:
I remove this manually. Not sure if this file is generated somehow

Reply:
No, that's completely safe. This was added by hand